src/chart/XYAxis.js

```diff
@@ -37,6 +43,11 @@
                 return context.brushMoved();
             })
         ;
+        this.yBrush2 = d3.svg.brush()
+            .on("brush", function () {
+                return context.brushMoved();
+            })
+        ;
```

Comment:
There should be no need for a new brush - you can trap a new ```.on("brush.y2", ...)``` event handler to the existing brush and do the selection handling there.